auslib/web/public/client.py

```diff
@@ -29,6 +29,33 @@ def getHeaderArchitecture(buildTarget, ua):
         return 'Intel'
 
 
+def getSystemCapabilities(systemCapabilities):
```

Comment:
Ah, OK, I thought that memory seemed a special case of instruction set, something like "an hardware with more than 8GB supports 64bits version".